paymentwall/widget.py

```diff
@@ -73,6 +74,46 @@ def get_params(self):
 				else:
 					self.append_to_errors('Only 1 product is allowed')
 
+		elif self.get_api_type() == self.API_CHECKOUT:
```

Comment:
This part is duplicated from `get_params()`. If you can refactored them into 1 single method, that'd be great